src/storageDataGroup.js

```diff
@@ -489,11 +456,11 @@ class StorageDataGroup {
                 };
                 const payload = { _taskState: 'VALIDATING' };
 
-                return iControlRestRequest(opts, payload);
+                return request.send(opts, payload);
             })
-            .then((response) => {
-                if (response.status !== 202) {
-                    return Promise.reject(new Error(`failed to update save sys config task:${JSON.stringify(response.body)}`));
+            .then((res) => {
+                if (res.message !== 'Task will execute asynchronously') {
```

Comment:
There's no way to get the HTTP status?

Reply:
I'm looking through atg-shared-utilities and it looks like all it returns is the body. We'd need to update atg-shared-utilities and change the response object to include meta data. In this particular instance we do get the following as the response:

```
{ code: 202,
  message: 'Task will execute asynchronously',
  errorStack: [] }

```

Instead of using the message, I would rather use the code of 202. What do you think? :)

Reply:
Yes, I definitely prefer the code.